gemac/client.py

```diff
@@ -0,0 +1,30 @@
+from myhdl import block, instances
+
+
+@block
+def client(gtxClk, txData, txDataValid, txIFG_delay,
```

Comment:
review the PEP8 naming conventions, CamelCase is typically reserved for class names.

Also, utilize [interfaces](http://docs.myhdl.org/en/stable/whatsnew/0.9.html#interfaces-conversion-of-attribute-accesses)